doc/usecases/layers/photo_layer.md

```diff
@@ -0,0 +1,27 @@
+# Use Case: Photo Layer
+
+## Summary
+
+- **Scope:** Photo Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can add, move and delete pictures in a map in the photo layer.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  The user has opened the app and has navigated to a map.
+- **Main success scenario:**
+  - The user successfully adds, moves and deletes pictures in a map using the photo layer.
+  - When adding a picture the user can choose to upload one or choose from a preselection (emoticons, icons, etc..).
+  - When the user clicks on a picture a chat shows up where the photo can be discussed.
```

Comment:
```suggestion
  - When the user clicks on a picture a chat shows up where the photo can be discussed and commented (e.g. with thumbs up 👍).
```

Reply:
This may go beyond the scope of this use case alone, but maybe we could implement a kind of heat map to visualize which parts of the map were especially liked by other users, based on the number of positive emoticons.
I may also use the average 'thumbs-up-ratio' of the map to influence the map wide like counter that I propose in my use case.

Reply:
I think this is too much. What is a good idea is that people can like (etc.) the same place, without adding an additional picture. The sentence should be clarified to express this more clearly?

For the gamification an overall count of likes (summing up all the emoticons) seems to be enough.